apps/web/src/pages/Users/UserInformationPage/components/AboutSection.tsx

```diff
@@ -32,9 +32,9 @@ const AboutSection = ({ user }: BasicUserInformationProps) => {
         <div data-h2-flex-item="base(1of1) p-tablet(1of2) desktop(1of3)">
           <p data-h2-font-weight="base(700)">
             {intl.formatMessage({
-              defaultMessage: "Email:",
-              id: "3UkIDo",
-              description: "Display text for the email field on users",
+              defaultMessage: "Contact email address:",
+              id: "83ujdI",
+              description: "Display text for the contact email field on users",
```

Comment:
Can we use `formatMessage(commonMessages.email) + formatMessage(commonMessages.dividingColon)` here?

Reply:
It would leave a random trailing space but I could do that if you'd prefer.

Reply:
This would leave a trailing space? I just tried it and I don't see one 

```tsx
          <p data-h2-font-weight="base(700)">
            {intl.formatMessage(commonMessages.email) +
              intl.formatMessage(commonMessages.dividingColon)}
          </p>
```

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/f5f7cf1c-8372-4e66-a5f2-38ec1ce82258)
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/a7cbf2e9-8df4-4019-844a-d59958ca5a90)

Reply:
Nice thanks :+1:

Reply:
I was just talking about this.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/49166751/b89dad88-4df9-46f1-9a45-9fd47d9704a8)
It's really not an issue though so I've updated it in [c24cfca](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10855/commits/c24cfcaede1a8ee873ccec38202f9614ff39ceaa).